.travis.yml

```diff
@@ -1,7 +1,7 @@
-language: generic
-addons:
-  apt:
-    packages:
-      - parallel
-      - grep # Untested. Used with `ggrep` command in script, might need change
-script: /bin/bash ./scripts/health_check_links.sh
+sudo: required
+
+services:
+  - docker
+
+script:
+  - docker run -ti --rm -v $PWD:/mnt:ro simeg/urlsup `find . -name "*.md"` --threads 10 --allow 429 --white-list http://localhost
```

Comment:
should we allowing redirects (301 and 302) as well? Since most of these requests are redirects or http upgrades, i think they should be added as well

Reply:
Redirects could be useful to fail on as we could then update the links in the guide. WDYT?

Reply:
The current travis build does show 301s for valid URLs as well.. so we can test run this and then see if it breaks? I am fine with either

Reply:
Currently there are no redirects happening (I tried running this locally) so I'm fine with not allowing it for now. It's easy to change anyway!